Library/Formula/mdocml.rb

```diff
@@ -0,0 +1,23 @@
+class Mdocml < Formula
+  homepage "http://mdocml.bsd.lv/"
+  url "http://mdocml.bsd.lv/snapshots/mdocml-1.13.2.tar.gz"
+  sha1 "0d3c4e72214f73ee81c02e9b8863db0bc8f85aaf"
+
+  keg_only :provided_by_osx, <<-EOS.undent
+    In addition it may conflict with the groff formula from the homebrew-dupes tap.
+  EOS
+  
+  def install
+    ENV.deparallelize
```

Comment:
Doing a `make -j$(sysctl -n hw.ncpu) install` in the source tree of mdocml (CVS) works just fine.
What does homebrew exactly do when building in parallel?
I'd like to reproduce the error in the mdocml source tree before reporting upstream.